### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by lvh**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
   :dependencies [[org.clojure/clojure "1.8.0"]
-                 [manifold "0.1.4"]
+                 [manifold "0.1.6"]
                  [org.clojure/math.numeric-tower "0.0.4"]]
-  :plugins [[lein-auto "0.1.2"]
-            [lein-cljfmt "0.3.0"]
-            [lein-pprint "1.1.1"]
-            [lein-environ "1.0.2"]
+  :plugins [[lein-auto "0.1.3"]
+            [lein-cljfmt "0.5.6"]
```
Comment (ehashman) [inline project.clj]: cljfmt 0.5.6 has an incompatibility with lein-ancient iirc which is causing Travis to explode

Comment (ehashman) [reply project.clj]: (I'm happy with removing lein-ancient as a plugin here and letting users add it in their own profiles)

Comment (lvh) [reply project.clj]: Huh, crud. I thought that one was fixed in lein ancient >= 0.6.2. Hm.

**Comment by lvh** [comment]
Woo! It's probably still a clj-rewrite issue, but no longer one induced by lein-ancient I guess.

**Comment by ehashman** [comment]
Argh. Well, feel free to downgrade cljfmt, I don't mind.

**Comment by lvh** [comment]
🥇

**Review by ehashman — APPROVED** [review]

**Pull request merged**